dart/lib/sentry.dart

```diff
@@ -22,6 +22,7 @@ export 'src/transport/transport.dart';
 export 'src/integration.dart';
 export 'src/event_processor.dart';
 export 'src/http_client/sentry_http_client.dart';
+export 'src/http_client/sentry_http_client_error.dart';
```

Comment:
This is exported in order to be used by `sentry_dio`s `FailedRequestClientAdapter`.